### ISSUE TITLE
Fortnight how to run a Job every Fortnight , It's every second SAT. or /14 days. 

### ISSUE THREAD

**Issue by georgioua** [HangfireIO/Cronos#63]
Fortnight how to run a Job every Fortnight , It's every second SAT. or /14 days.

**Comment by aravind-alapati** [comment]
This `0 0 *   * 6#2` should work, reference: https://github.com/HangfireIO/Cronos#cron-format

**Comment by georgioua** [comment]
6=SAT
Is SUN=0 or 7? 
MON=1 ?

**Comment by aravind-alapati** [comment]
Both 0 and 7 means SUN
and MON =1 

![image](https://github.com/HangfireIO/Cronos/assets/126249156/d53e4f77-0dee-42d8-965e-0541e00602cc)

**Comment by georgioua** [comment]
This is not fortnight still. It's every second Saturday of the month.

**Comment by anvaravind** [comment]
![image](https://github.com/HangfireIO/Cronos/assets/12210689/bc2c93de-baac-489c-9a63-b72161343572)
 Isn't it what you asked for ?

**Comment by aravind-alapati** [comment]
Isn't it what you asked for ?
![image](https://github.com/HangfireIO/Cronos/assets/126249156/e4eae110-76eb-4135-ba4b-d657b0419038)

**Comment by christianhsmebye** [comment]
Have the same problem here.
I want to run a job every second friday. The Cron expression "0 0 * * 5#2,5#4" will run every other Friday, that is, the second and fourth Friday of each month. Starting from today (May 3, 2024), the next runs in 2024 will be:
May: 10th (2nd Friday) and 24th (4th Friday)
June: 14th (2nd Friday) and 28th (4th Friday)
July: 12th (2nd Friday) and 26th (4th Friday)
August: 9th (2nd Friday) and 23rd (4th Friday)

The **problem** here is that there is not every second friday:
May: 10th and 24th
June: 7th and 21th
July: 5th and 19th
August: 2th, 16th and 30th.

May is OK, but from June... not OK.

So, the feature request here is a kind of "skip". Ex: skip 0 means every occation, skip 1 means that you skip one occation, skip 2 means that you skip two occation etc.

This would be very helpful! Thanks!

**Comment by JohnSpikeKelly2** [comment]
Yes, I believe more recent cron implementations will support 0 0 */14 * * and this will be every 14 days. However, this is currently interpreted as the 1st, 15th and 29th of each month.

Changing the behavior to be every 14 days is a fundamental change in how cron works, it needs to know every 14 days from a specific start date, which today it does not need to know a reference date, it can determine all dates without a reference date.

Anyways, I think adding the ability would be useful.

One suggestion would be to combine it with a specific day of the week. Example:
0 0 */14 * * would be 1st, 15th, 29th of the month, whereas
0 0 */14 * MON would be Every second Monday, or possibly
0 0 * * MON/2 could also be Every second Monday
